src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -863,7 +966,27 @@ def _visit_generic_builtin(self, node: gtir.FunCall) -> DataExpr:
                     connector,
                 )
 
-        return self._construct_tasklet_result(dtype, tasklet_node, "result")
+        if isinstance(node.type, itir_ts.ListType):
+            # The only builtin function (so far) handled here that returns a list
+            # is 'make_const_list'. There are other builtin functions (map_, neighbors)
+            # that return a list but they are handled in specialized visit methods.
+            # This method (the generic visitor for builtin functions) always returns
+            # scalars. This is also the case of 'make_const_list' expression: it simply
+            # broadcasts a scalar value on the local domain of another expression,
+            # for example 'map_(plus)(neighbors(V2Eₒ, it), make_const_list(1.0))'.
```

Comment:
Just for my understanding, what does this expression do?
Does it add 1.0 to every first edge enighbor of the iterator `it`?
Furthermore, since this is the fieldview representation, why are you still using an iterator?